gateway/modules/auth/match.go

```diff
@@ -14,31 +14,29 @@ import (
 	"github.com/spaceuptech/space-cloud/gateway/config"
 	"github.com/spaceuptech/space-cloud/gateway/model"
 	"github.com/spaceuptech/space-cloud/gateway/utils"
-
-	"github.com/spaceuptech/space-cloud/gateway/modules/crud"
 )
 
-func (m *Module) matchRule(ctx context.Context, project string, rule *config.Rule, args, auth map[string]interface{}) (*PostProcess, error) {
+func (m *Module) matchRule(ctx context.Context, project string, rule *config.Rule, args, auth map[string]interface{}) (*model.PostProcess, error) {
```

Comment:
do the same change in this file replace with nil

Reply:
change at all occurances in the file?